source/image/Sprite.cpp

```diff
@@ -20,6 +20,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include "../Screen.h"
 
 #include "../opengl.h"
+
```

Comment:
```suggestion
```

Reply:
I was responding to your code clean-up comment.  Otherwise I never would have touched this file, but there was a Warning in there saying there was a missing blank line.  So I added it.

You really want to put the warning back?

Reply:
Alright, I thought it was accidental. But if you want your PR (which has already more than 1k lines) reviewed and merged sooner than later, don't include too much unrelated changes.